OpenRA.Mods.Common/Traits/SpawnActorOnDeath.cs

```diff
@@ -50,6 +50,9 @@ public class SpawnActorOnDeathInfo : ITraitInfo
 			"lead to unexpected behaviour.")]
 		public readonly CVec Offset = CVec.Zero;
 
+		[Desc("Defines if an actor is spawned even after the player was defeated (e.g. after surrendering).")]
+		public readonly bool SpawnAfterGameEnd = true;
```

Comment:
Well, the game ended for this one player. :P
I just didn't want to go with `SpawnAfterDeath` (death of the player), as that is kind of ambiguous. Now that I think of it again, what about `SpawnAfterDefeat`?